cve_bin_tool/extractor.py

```diff
@@ -86,7 +106,21 @@ def extract_file_rpm(cls, filename, extraction_path):
     def extract_file_deb(cls, filename, extraction_path):
         """ Extract debian packages """
         if not inpath("ar"):
-            print("Error: need 'ar' binary in path")
+            """ Acutally MinGW provides ar, so this might never get called """
+            import unix_ar
```

Comment:
You'll want to add:

```python
  install_requires=[
    "unix-ar>=0.2.1",
  ],
```

to setup.py

Reply:
should I also make changes to the travis configuration file? The thing is that `pip install unix_ar` only works for python3, for python2 it has to build from source. (by the way I actually think this is not necessary since `ar` is supported both on Linux and Windows, so I'm considering discard this python-implemented ar stuff)

Reply:
if ar's supported on both Linux and Windows, that's good enough for me for now.  We can revisit when we drop python2 support in 2020.